client/components/editor/teaching-elements/Assessment/TextSelection/index.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div :class="{ disabled: !isEditing }">
+    <h5 class="title">Text and selection</h5>
+    <div
+      @click="focus"
+      class="container">
+      <te-html
+        v-if="!isSelecting && isEditing"
+        :element="textElement"
+        :is-focused="isFocused"
+        @save="saveContent"/>
+      <selector
+        v-else
+        :content="this.text"
+        :correct="correct"
+        :isEditing="isEditing"
+        @save="saveSelection"
+        class="htmlContent ql-container ql-snow"/>
+    </div>
+    <div v-if="isEditing">
+      <button
+        v-if="isSelecting"
+        @click="edit"
+        class="btn btn-primary btn-material">
+        <span class="mdi mdi-stop-circle-outline"/>
+          Edit content
+      </button>
+      <button
+        v-else
+        @click="select"
+        class="btn btn-primary btn-material">
+        <span class="mdi mdi-marker"/>
+          Select solutions
+      </button>
```

Comment:
Well, few issues:
1. This is toggle button which means it should be **one** button
2. `<span>` can't be selfclosed
3. Class `.icon` is missing here
4. Button labels are wrongly indented
5. Attribute `type="button"` is missing on `<button>` element